### PR TITLE
[FUCK] Forgotten Ship ruin fix 2: Policy config edition.

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

**First of all this PR adds config for ghost-role policy(#50185)**
**Also, it's going to fix and balance the forgotten ship ruin even more.**
**Cybersun Hardsuit armour values changed to:** _(**"melee" = 30**, **"bullet" = 40**, "laser" = 70, **"energy" = 70**, **"bomb" = 30**, "bio" = 100, **"rad" = 60**, **"fire" = 60**, "acid" = 60)_

## Why It's Good For The Game

In the current state, ruin may be disappointing and unbalanced.
Even though I'm not sure I can balance it to the ideal state, this fix also changes ruin's chance to spawn so we won't have it every round 24/7.
Furthermore, this PR adds Cybersun Hardsuit to traitors(14TC) and nuke ops(8TC) uplinks, so you don't have to ask admins for TC trade anymore.

## Changelog
:cl: EgorDinamit
add: Cybersun hardsuit added to traitors(14TC) and nuke ops(8TC) uplinks.
balance: Cybersun hardsuit armour values increased.
tweak: Adds and replaces a few items on the ship, such as firing pins, materials and encryption keys.
balance: Syndicate ship ruin turrets armour and rate of fire increased.
fix: Empty sleeper, that emits GPS signal is now indestructible.
config: Adds config to edit "important_info" for both, crewmembers and captain of forgotten ship ruin.
/:cl:

### CODE REVIEW

File: code/modules/ruins/spaceruin_code/forgottenship.dm [hunk 1]
```diff
@@ -33,18 +33,45 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
...
 
 /obj/structure/fluff/empty_sleeper/syndicate/captain/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/gps, "Old Encrypted Signal")
 
+/obj/item/storage/box/firingpins/syndicate
+	name = "box of syndicate firing pins"
+	desc = "A box full of special syndicate firing pins which allow only syndicate operatives to use weapons with those firing pins."
+	icon_state = "secbox"
+	illustration = "firingpin"
```
Comment (TiviPlus) [inline code/modules/ruins/spaceruin_code/forgottenship.dm]:
```suggestion
```

**Comment by Fikou** [comment]
the cybersun hardsuit now has 20 more laser armor than what's supposed to be the best anti laser armor in the game

File: code/modules/ruins/spaceruin_code/forgottenship.dm (hunk 1 above)
Comment (EgorDinamit) [reply code/modules/ruins/spaceruin_code/forgottenship.dm]: Oh, the parent object has those already, right?

Comment (TiviPlus) [reply code/modules/ruins/spaceruin_code/forgottenship.dm]: It does

**Comment by EgorDinamit** [comment]
> the cybersun hardsuit now has 20 more laser armor than what's supposed to be the best anti laser armor in the game

60% armour + 50% chance to reflect lasers is still better, especially if you take in mind that disabler beams stamina damage isn't reduced by armour.

Let's just say, someone makes 4 shots with a laser gun(20 damage):
Cybersun: _Reduced from 80 to 16._
Reflector vest: [2 shots reflected, took only 2 shots instead of 4] _Reduced from 80(40) to 16._
So technically, they are somewhat the same, except cybersun one is a hardsuit.

**Comment by EgorDinamit** [comment]
Anyway, @Fikou , that's why we have an open conversation here - to share ideas.

**Comment by spookydonut** [comment]
You need to fight against #50171, only one can live.

**Comment by Timonkeyn** [comment]
i dont support this as long as there is no policy regarding the syndies

**Comment by EgorDinamit** [comment]
> i dont support this as long as there is no policy regarding the syndies

I pinged some headmins about this on discord, soon enough there will be one.
Basically it means that syndies can do anything except for invading station/lavaland.

**Comment by EgorDinamit** [comment]
Well then, I guess I'll change protection values to this instead:
**("melee" = 30, "bullet" = 40, "laser" = 70, "energy" = 50, "bomb" = 30, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 60)**

**Comment by EgorDinamit** [comment]
AppVeyor is kil, someone restart it later.

**Comment by IndieanaJones** [comment]
I still this runs into the same problems as most ghost roles do, in that there's an entire lack of replayability on all fronts.  You get to sit in a ship that can't move all round, and very, very, rarely the outside world might come try to kill you.  Once you do this ruin once or twice, you have no reason to do it again.

**Comment by EgorDinamit** [comment]
@IndieanaJones well, that's a ghost-roles, they aren't supposed to be full-time jobs, even though playing HoP or any other role for 100 hours might get boring at some point.
How fun it will be depends on the current round. 
**You got nuke ops?**: Why not build a teleporter and throw them some cool stuff(not people/borgs) to help them.
**You got smart traitors?**_(They bought encryption key)_: Help them navigate to the ship or if they have access to teleporter - print a beacon for a moment for them. It's even better if you got some syndie-ships and someone built gigabeacon on the station.
**Green shift?**: Green shift often means total boredom/autistic projects. That probably will double the chances for someone saying "Hey, let's raid this shit in space" or whatever.

At the same time, Charlie station is just shit. You might build a singulo engine, but I highly doubt that someone can do this without destroying everything in a 1-kilometre radius. Of course, you still can go to the station and, I dunno, be a wandering merchant? Ask HoP for a normal job here? Probably you'll just end up as "Someone who has omnilathe" for science or shady persons.

**Comment by EgorDinamit** [comment]
Oh god oh fuck

**Comment by EgorDinamit** [comment]
Shit's fucked

**Comment by EgorDinamit** [comment]
I'll open a new one, *sigh*

**Pull request closed without merging**